### DISCUSSION TITLE
Make tiny birds have no foot steps

### DISCUSSION THREAD

**Discussion by AzureMH** [aqariio/Fowl-Play#27] [Ideas]
So i think it would be better if tiny birds have no foot steps, cuz its scary to hear foot steps around my base and i always feel that if theres a mob around.

**Comment by AzureMH** [comment]
Plus i wonder if theres a tutorial for this mod?

**Reply by aqariio** [reply]
The mod has a wiki, but it's currently incomplete. I'll work on expanding it soon, but in the meantime you can look through it. The wiki currently has some basic information regarding where the birds spawn and their mechanics.

**Reply by AzureMH** [reply]
I see

**Comment by aqariio** [comment]
I'll fix this in the next update

**Reply by AzureMH** [reply]
Thanks for the update! It's so awwwwwwwesome!
